modules/cli/src/test/java/org/apache/ignite/internal/cli/commands/treesitter/highlighter/HighlighterTest.java

```diff
@@ -21,34 +21,34 @@
 import static org.junit.jupiter.params.provider.Arguments.of;
 
 import java.util.stream.Stream;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
 
 /**
  * Tests for the highlighting errors in JSON text. The colors itself are not tested.
  */
-class JsonAnsiHighlighterTest {
-    JsonAnsiHighlighter highlighter;
+class HighlighterTest {
 
-    private static Stream<Arguments> textInput() {
+    private static Stream<Arguments> jsonTextInput() {
         return Stream.of(
                 of("plain", "plain"),
                 of("{\"key\": \"val\"}", "{\"key\": \"val\"}"),
                 of("{\"key\": \"1%\"}", "{\"key\": \"1%\"}")
         );
     }
 
-    @BeforeEach
-    void setUp() {
-        highlighter = new JsonAnsiHighlighter();
+    @ParameterizedTest
+    @MethodSource("jsonTextInput")
+    void jsonHighlight(String input, String expected) {
+        String result = JsonAnsiHighlighter.highlight(input);
+        assertEquals(expected, result);
     }
 
     @ParameterizedTest
-    @MethodSource("textInput")
-    void testHighlight(String input, String expected) {
-        String result = highlighter.highlight(input);
+    @MethodSource("jsonTextInput")
+    void hoconHighlight(String input, String expected) {
```

Comment:
can you add one more MethodSource for hocon test case? It would be nice to test not only json input but hocon too.